guacamole-ext/src/main/java/org/apache/guacamole/net/auth/simple/SimpleConnection.java

```diff
@@ -157,6 +162,43 @@ public SimpleConnection(String name, String identifier,
         this.fullConfig = config;
         this.interpretTokens = interpretTokens;
 
+    }
+    
+        /**
+     * Creates a new SimpleConnection having the given identifier,
+     * GuacamoleConfiguration, and GuacamoleProxyConfiguration. Parameter tokens
+     * will be interpreted if explicitly requested.
+     *
+     * @param name
+     *     The name to associate with this connection.
+     *
+     * @param identifier
+     *     The identifier to associate with this connection.
+     *
+     * @param config
+     *     The configuration describing how to connect to this connection.
+     *
+     * @param interpretTokens
+     *     Whether parameter tokens in the underlying GuacamoleConfiguration
+     *     should be automatically applied upon connecting. If false, parameter
+     *     tokens will not be interpreted at all.
+     * 
+     * @param proxyConfig
+     *     The Guacamole proxy configuration used to tell the client how to
+     *     connect to guacd.
+     */
+    public SimpleConnection(String name, String identifier,
+            GuacamoleConfiguration config, boolean interpretTokens,
+            GuacamoleProxyConfiguration proxyConfig) {
```

Comment:
Thoughts on parameter order here?

To me, something like:

```java
    public SimpleConnection(String name, String identifier,
            GuacamoleProxyConfiguration proxyConfig
            GuacamoleConfiguration config, boolean interpretTokens) {
```

feels more predictable and logical.

Reply:
I'm good with that.